error-prone-contrib/src/test/java/tech/picnic/errorprone/bugpatterns/util/MoreMatchersTest.java

```diff
@@ -47,9 +50,33 @@ void hasMetaAnnotation() {
         .doTest();
   }
 
-  /** A {@link BugChecker} that delegates to {@link MoreMatchers#hasMetaAnnotation(String)} . */
+  @Test
+  void hasLombokDataAnnotation() {
```

Comment:
Given the the constant field is listed first, I'd move this method up (and likewise for the test checker definition below).